corehq/apps/locations/migrations/0012_adding_indices_for_warehouse.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.8 on 2018-01-30 10:56
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
It might be worth moving this into a branch and PR against master itself to avoid potential migration conflicts since this PR might sit here for a long time? (unless this depends on any of pg9.6 or warehosue work)